WorldOfPAYBACK/View/TransactionsDetailView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  TransactionsDetailView.swift
+//  WorldOfPAYBACK
+//
+//  Created by Kacper Rubaszewski on 20/02/2023.
+//
+
+import SwiftUI
+
+
+struct DetailsViewLoading: View {
+    
+    @Binding var transaction: Item?
+
+    var body: some View {
+
+        if let transaction = transaction {
```

Comment:
hint: I think you can now just write `if let transaction` or something like this :)